src/Cms/Languages.php

```diff
@@ -45,11 +46,17 @@ public function create(array $props)
      */
     public function default()
     {
-        if ($language = $this->findBy('isDefault', true)) {
-            return $language;
-        } else {
-            return $this->first();
+        if ($defaults = $this->filterBy('isDefault', true)) {
+            if ($defaults->count() === 1) {
+                return $defaults->first();
+            }
+
+            if ($defaults->count() > 1) {
+                throw new DuplicateException('You cannot have two default languages. Please check your language config files.');
```

Comment:
Maybe "You cannot have multiple default languages" to cover all cases in the error message.